cmd/svcat/test/helper.go

```diff
@@ -38,3 +40,27 @@ func NewContext(outputCapture io.Writer, fakeApp *svcat.App) *command.Context {
 		App:    fakeApp,
 	}
 }
+
+// OutputMatches compares expected output, optionally allowing for different line ordering.
+func OutputMatches(gotOutput string, wantOutput string, allowDifferentLineOrder bool) bool {
+	if !allowDifferentLineOrder {
+		return strings.Contains(gotOutput, wantOutput)
+	}
+
+	gotLines := strings.Split(gotOutput, "\n")
+	wantLines := strings.Split(wantOutput, "\n")
+
+	for _, wantLine := range wantLines {
+		found := false
+		for _, gotLine := range gotLines {
+			if strings.Contains(gotLine, wantLine) {
+				found = true
+				break
```

Comment:
I'm thinking we should delete found lines from the gotOutput when we get a match

Reply:
Since I didn't modify the existing implementation of this function (just moved it to reuse it) do you mind if I do that in a follow-up PR?

Reply:
yeah, that is fine by me, I'm not even convinced it's necessary.